### PR TITLE
Add CtlplaneNetworkAttribues for 16.2

### PR DESCRIPTION
**Pull request by olliewalsh**
Tripleoclient now sets this in stable/train - https://review.opendev.org/c/openstack/python-tripleoclient/+/825689

### CODE REVIEW

File: templates/openstackconfiggenerator/config/16.2/rendered-tripleo-config.yaml [hunk 1]
```diff
@@ -123,4 +123,38 @@ parameter_defaults:
...
 {{- end }}
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+  #
+  # CtlplaneNetworkAttributes
+  CtlplaneNetworkAttributes:
+    {{- range $netname, $net := .NetworksMap }}
+    {{- $netname_lower := lower $net.Name }}
```
Comment (stuggi) [inline templates/openstackconfiggenerator/config/16.2/rendered-tripleo-config.yaml:131]: netname_lower is not used and should be not required as there is $net.NameLower? Think we can also remove it from the 17.0 templates

Comment (olliewalsh) [reply templates/openstackconfiggenerator/config/16.2/rendered-tripleo-config.yaml]: done

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**